sw/ground_segment/python/natnet3.x/NatNetClient.py

```diff
@@ -235,15 +236,15 @@ def __unpackMocapData( self, data ):
                 offset += 4
 
                 # Version 2.6 and later
-                if( ( self.__natNetStreamVersion[0] == 2 and self.__natNetStreamVersion[1] >= 6 ) or self.__natNetStreamVersion[0] > 2 or major == 0 ):
+                if( ( self.__natNetStreamVersion[0] == 2 and self.__natNetStreamVersion[1] >= 6 ) or self.__natNetStreamVersion[0] > 2):
```

Comment:
it actually seems that these `or major == 0` are a bad copy/paste from the C/C++ example. It should have been `or self.__natNetStreamVersion[0] == 0`. I don't know if we should keep them or not.